src/emucore/Settings.cxx

```diff
@@ -672,3 +688,59 @@ Settings& Settings::operator = (const Settings&)
 
   return *this;
 }
+
+// - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+void Settings::setDefaultSettings() {
+
+    // Stella settings
+    stringSettings.insert(pair<string, string>("cpu", "low")); // Reduce CPU emulation fidelity for speed
```

Comment:
Having one container for each setting type is a bit clunky, but it is a nice safe option. We could consider hiding the multiplicity of containers behind a templated method which automatically returns the correct container, but this will do too.